plugins/panel-examples/__init__.py

```diff
@@ -0,0 +1,940 @@
+            if sample.ground_truth.detections is not None:
+                for detection in sample.ground_truth.detections:
+                    label = detection.label
+                    if label not in label_counts:
+                        label_counts[label] = 1
+                    else:
+                        label_counts[label] += 1
+
+        # sort label counts by values and create list of only the keys of label counts in descending order
+        sorted_label_counts = sorted(
+            label_counts.items(), key=lambda x: x[1], reverse=True
+        )
+        labels = [label_count[0] for label_count in sorted_label_counts]
+        values = [label_count[1] for label_count in sorted_label_counts]
+
+        ctx.panel.state.labels = labels
+        ctx.panel.state.values = values
+
+        histogram_data = [
+            {
+                "x": ctx.panel.state.labels,
+                "y": ctx.panel.state.values,
+                "type": "bar",
+                "marker": {"color": "orange"},
+            }
+        ]
+        layout = {
+            "width": 600,
+            "xaxis": {"title": "Label Name"},
+            "yaxis": {"title": "Count"},
+            "title": "A Fancy Plot",
+        }
+
+        ctx.panel.state.histogram = histogram_data
+        ctx.panel.state.layout = layout
+
+    @staticmethod
+    def on_change_ctx(ctx: ExecutionContext):
+        # trigger actions when any change happens within fiftyone server
+        pass
+
+    @staticmethod
+    def on_change_view(ctx: ExecutionContext):
+        # trigger actions when any change happens within fiftyone view such as sample filtering
+        pass
+
+    def filter_data(self, ctx: ExecutionContext):
+        filter_label = ctx.params["data"]["label"]
+        # create a view of the dataset that only includes samples with the filter label
+        filtered_view = ctx.dataset.filter_labels(
+            "ground_truth", F("label") == filter_label
+        )
+        # display the filtered view
+        ctx.trigger(
+            "set_view",
+            params=dict(
+                view=json.loads(json_util.dumps(filtered_view._serialize()))
+            ),
+        )
+
+    def reset(self, ctx: ExecutionContext):
+        ctx.ops.clear_view()
+        self.on_load(ctx)
+
+    def render(self, ctx: ExecutionContext):
+        panel = types.Object()
+
+        # Bar Chart - Histogram
+        panel.plot(
+            "histogram",
+            layout=ctx.panel.state.layout,
+            label="Interactive Histogram",
+            on_click=self.filter_data,
+        )
+
+        # Button
+        panel.btn("reset", label="Reset Chart", on_click=self.reset)
+
+        return types.Property(
+            panel,
+            view=types.GridView(
+                align_x="center", align_y="center", orientation="vertical"
+            ),
+        )
+
+
+###
+# Dropdown Menu
+###
+
+
+class DropdownMenuPanel(foo.Panel):
```

Comment:
Can you update this to show how to build a menu like the embedding panel's that contains a cluster of dropdown + actions in a `menu()`? Extremely useful pattern

Reply:
I'll check this out